### DISCUSSION TITLE
Getting 504 gateway timeout on my web application

### DISCUSSION THREAD

**Discussion by mvrceloz** [vercel/community#6043] [Help]
### Summary

Basically i have a web application and i'm getting the error: "504: gateway timeout" and similars since the day before yesterday.

i don't know what to do, the application was working fine last week, but now is breaking in each page.

![image](https://github.com/vercel/community/assets/47905839/a04b6361-c8fe-41d0-b3d3-1a779bb350d1)

when i go to see the logs, i just get something like: "timeout" or "prisma initialization error", but everything is fine, my backend and my database

![image](https://github.com/vercel/community/assets/47905839/2f349819-77df-4e18-9fcb-3bc2ac4d295d)

my website are all hosted on vercel, and my database and backend are on brazil, websites that uses a north america database or backend are working fine, but why????

### Steps to Reproduce

to reproduce you must try to create an database in brazil and host an front end connecting to it on vercel

**Comment by amyegan** [comment]
Hey @mvrcelitos. A timeout error can sometimes be caused by invalid access tokens. It would be worth checking whether a token is expired that could be preventing database connection.
